src/main/java/net/minecraftforge/fluids/FluidStack.java

```diff
@@ -123,14 +124,14 @@ public final Fluid getFluid()
         return fluidDelegate.get();
     }
 
-    public String getLocalizedName()
+    public String getDisplayName()
```

Comment:
Would it make more sense to have this be an `ITextComponent` like `Item#getDisplayName(ItemStack stack)` is?

Reply:
Done. Good idea.